### DISCUSSION TITLE
大佬你好，请问这个mumupro破解工具怎么使用？

### DISCUSSION THREAD

**Discussion by ypscarab** [lisai9093/YYS#16] [Q&A] [answered]
大佬你好，请问这个mumupro破解工具怎么使用？

**Comment by lisai9093** [comment, marked as answer]
这个是我在隔壁论坛看到的：https://xdaforums.com/t/android-on-macos-with-appale-chip-mumuplayer-cracked.4661957/
使用方法就是终端先安装证书：
`sudo ./main install`
之后运行服务器即可：
`sudo ./main`

**Reply by lisai9093** [reply]
个人实测1.3.6版本及以下才能使用，新版本用不了
